level/level.go

```diff
@@ -1,6 +1,15 @@
 package level
 
-import "github.com/go-kit/log"
+import (
+	"errors"
+	"strings"
+
+	"github.com/go-kit/log"
+)
+
+var (
+	ErrInvalidLevelString = errors.New("invalid level string")
+)
```

Comment:
Please add documentation for this exported error value. Also please write the declaration on a single line without the `()`'s.